linkerd/core/src/test/scala/io/buoyant/linkerd/RouterTest.scala

```diff
@@ -96,4 +102,23 @@ servers:
     val DstBindingFactory.Namer(interpreter) = router.params[DstBindingFactory.Namer]
     assert(interpreter.isInstanceOf[TestInterpreter])
   }
+
+  test("with retries") {
+    val yaml =
+      """|protocol: plain
+         |client:
+         |  retries:
+         |    backoff:
+         |      kind: jittered
+         |      maxMs: 10000
```

Comment:
This test is a bit confusing.  It creates a RetriesConfig that will fail if `.mk` is called on it.  Since you test that error case elsewhere, I'd just give a fully valid config here.